content/episodes/113-multimedia-no-linux-part-2.rst

```diff
@@ -0,0 +1,176 @@
+  softwares de áudio que ele já utilizou no Linux
+* Uma comparação entre `Ardour`_ e `Audacity`_
+* Sobre a `Linguagem ChucK`_
+* Quais são as vantagens (e desvantagens) no trabalho com áudio no Linux
+* O que precisa para ligar uma mesa de som no Linux
+* Sobre a biblioteca escrita em Python chamada AudioLazy
+
+Como sempre, nossa entrevista foi gravada ao vivo no nossa canal no `YouTube`_ e para ver a
+gravação deste episódio confira o vídeo abaixo:
+
+.. youtube:: 57f3G0A6cnM
+
+A `RubyConf <http://eventos.locaweb.com.br/proximos-eventos/rubyconf-2017/>`_
+em parceria com o Castálio Podcast disponibilizou 10 ingressos para sortear
+para nossos ouvintes e também o cupom **PRC_Castalio** que concede 30% de
+desconto na compra do ingresso, à partir do segundo lote. Serão sorteados dois
+ingressos por episódio e para concorrer basta preencher o `formulário
+<http://bit.ly/CastalioRubyConf>`_ e estar inscrito no `canal do YouTube
+<http://www.youtube.com/c/CastalioPodcast>`_.
+
+Antes de finalizar o post, não podemos deixar de agradecer a todos que nos
+deixaram comentários. Se você tem algo a nos dizer, você pode deixar seus
+comentários aqui no site, ou no `Twitter <https://twitter.com/castaliopod>`_ ou
+no `Facebook <https://www.facebook.com/castaliopod>`_. Também não esqueça de
+seguir a `Playlist do Castálio Podcast
+<https://open.spotify.com/user/elyezermr/playlist/0PDXXZRXbJNTPVSnopiMXg>`_ e
+assinar nosso `canal do YouTube <http://www.youtube.com/c/CastalioPodcast>`_.
+
+Até o próximo episódio!
+
+Contato
+-------
+
+.. raw:: html
+
+    <div class="row">
+        <div class="col-md-6">
+            <p>
+            <div class="media">
+            <div class="media-left">
+                <img class="media-object img-circle img-thumbnail" src="/images/danilo-bellini.jpg" alt="Danilo J. S. Bellini" width="200px">
+            </div>
+            <div class="media-body">
+                <h4 class="media-heading">Danilo J. S. Bellini</h4>
+                <ul class="list-unstyled">
+                    <li><i class="fa fa-github"></i> <a href="https://github.com/danilobellini">Github</a></li>
+                    <li><i class="fa fa-bitbucket"></i> <a href="https://bitbucket.org/danilobellini/">BitBucket</a></li>
+                    <li><i class="fa fa-twitter"></i> <a href="https://twitter.com/danilobellini">Twitter</a></li>
+                    <li><i class="fa fa-facebook"></i> <a href="https://www.facebook.com/djsbellini">Facebook</a></li>
+                    <li><i class="fa fa-linkedin"></i> <a href="https://www.linkedin.com/in/danilo-j-s-bellini-66a96310/">LinkedIn</a></li>
+                    <li><i class="fa fa-slideshare"></i> <a href="http://slideshare.net/djsbellini/">SlideShare</a></li>
+                </ul>
+            </div>
+            </div>
+            </p>
+        </div>
+    </div>
+
+Escute Agora
+------------
+
+.. podcast:: castalio-podcast-113
+
+Top 5
+-----
+
+* **Música**: `War Eternal`_ por `Arch Enemy`_
```

Comment:
Gostei de separar a musica so artista, antes eu colocava o link para musica direto o que tornava implicito o artista. Ja ouvi por ai que "Explicit is better than implicit." :smile: